sidebars.json

```diff
@@ -916,6 +916,42 @@
           "id": "getting-started/tutorials/project-template-integrated-with-jenkins",
           "type": "doc"
         },
+        {
+          "label": "AuthN/AuthZ Architecture",
+          "type": "category",
+          "collapsed": true,
+          "link": {
+            "type": "generated-index",
+            "title": "AuthN/AuthZ Architecture ",
```

Comment:
Can't we keep using the Auth contraction and have `Auth Architecture` menu title? instead of the one proposed here, it feels a bit complex and hard to read title 🤔

Reply:
I agree. I was very uncertain about the title because on one hand I wanted to include both authentication and authorization on the other hand, as you also suggest, the Auth contraction is much more readable. I have changed it in `Auth Architecture`